src/ConnectFour/Port/Adapter/Persistence/Repository/PredisGameStore.php

```diff
@@ -9,15 +9,22 @@
 use Gaming\ConnectFour\Application\Game\Query\Model\Game\GameFinder;
 use Gaming\ConnectFour\Application\Game\Query\Model\Game\GameStore;
 use Gaming\ConnectFour\Domain\Game\GameId;
-use Predis\ClientInterface;
+use Predis\Client;
+use Predis\ClientContextInterface;
 
 final class PredisGameStore implements GameStore
 {
+    /**
+     * @var array<string, Game>
+     */
+    private array $pendingGames = [];
+
     public function __construct(
-        private readonly ClientInterface $predis,
+        private readonly Client $predis,
```

Comment:
Switching from interface to concrete because `pipeline()` is unfortunately not part of the interface.